CMakeLists.txt

```diff
@@ -894,6 +901,11 @@ elseif (MSVC)
   set(CMAKE_CXX_FLAGS_RELEASE        "/MT /O2 /Ob2"                      CACHE INTERNAL "C++ release flags")
   set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "/MT /Zi /O2 /Ob1"                  CACHE INTERNAL "C++ release with debug info flags")
 
+  if (USE_CLCACHE_MODE)
```

Comment:
shouldn't this also replace all the Zi's above?

Reply:
Yes. It will replace all to Z7 for CLCACHE.